skt/executable.py

```diff
@@ -163,10 +175,21 @@ def cmd_merge(cfg):
             (retcode, _) = ktree.merge_git_ref(*mb)
 
             utypes.append("[git]")
-            idx += 1
             if retcode:
+                if retcode != 2:
```

Comment:
@veruu Why dont' you use the string constant instead of these numbers. It is easier to read and makes more sense?

Reply:
@danghai good idea! I'll open an issue for it, since it's not really a part of this series.

Reply:
@veruu one more is why do not take both if condition into one. if `retcode !=0 and recode !=2`. Or you will add some thing in future?

Reply:
Two reasons - this allows us to implement more return codes if we need in the future and not change too many things around; and we want the `return` that is below to execute if any failure occurs, not only infrastructure error, so we can't really merge the conditions anyways.